joomla/joomla-cms
# [4.0] Change 'Menu Class Suffix' to 'Menu Class'

Pull Request for Issue ##16894 (related) .

### Summary of Changes
Changes 'Menu Class Suffix' to 'Menu Class'.


### Testing Instructions
Add class to menu module using the 'Menu Class' field (no space). Check frontend and ensure class has been added correctly to the ul tag.

`<ul class="nav flex-column yourClass">`

### Expected result



### Actual result



### Documentation Changes Required
Yes
